addons/mail/static/src/xml/activity.xml

```diff
@@ -9,7 +9,7 @@
                 <div class="o_thread_message_sidebar">
                     <img t-attf-src="/web/image/res.users/#{activity.user_id[0]}/image_small"
                         class="o_thread_message_avatar mb8"/>
-                    <a href="#" class="o_activity_done o_activity_link" t-att-data-activity-id="activity.id" data-toggle="popover"><span class="fa fa-check-circle fa-2x" /></a>
+                    <a href="#" class="o_activity_done o_activity_link" t-att-data-activity-id="activity.id" t-att-data-activity-type-id="activity.activity_type_id[0]" data-toggle="popover"><span class="fa fa-check-circle fa-2x" /></a>
```

Comment:
Rename to t-att-data-previous-activity-type-id . Update accordingly remaining code.